packages/lib/server-only/2fa/enable-2fa.ts

```diff
@@ -0,0 +1,67 @@
+import { TRPCError } from '@trpc/server';
+
+import { ErrorCode } from '@documenso/lib/next-auth/error-codes';
+import { prisma } from '@documenso/prisma';
+import { User } from '@documenso/prisma/client';
+
+import { verifyTwoFactor } from './verify-2fa';
+
+type enableTwoFactorAuthenticationOptions = {
+  user: User;
+  code: string;
+};
+
+export const enableTwoFactorAuthentication = async ({
+  user,
+  code,
+}: enableTwoFactorAuthenticationOptions) => {
+  const encryptionKey = process.env.DOCUMENSO_ENCRYPTION_KEY;
```

Comment:
Lines 18 to 32 have been re-used in 2 places already. If it is used in other places as well, should we extract it, or?